container_files/assets/www/doc/builtin/sql/ValueExpression.md

```diff
@@ -472,48 +476,60 @@ number of occurrences of those tokens within `str`. For example `tokenize('a b b
 
 ### JSON unpacking <a name="unpack_json"></a>
 
-The `unpack_json(str)` function will parse the string `str` as a JSON object and unpack it into multiple columns following the  algorithm outlined below. Note that JSON objects shown in the tables below are string reprensetations of the JSONs.
+The `unpack_json(str)` function will parse the string `str` as a JSON
```

Comment:
so `unpack_json` is still there but uses the same underlying implementation as `parse_json` and so has changed behaviour? I'm confused.